UPGRADE.md

```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`
+    - run `docker-compose up -d`
+    - create Elasticsearch indexes by running `./phing elasticsearch-indexes-create`

```

Comment:
please use `php phing` instead of `./phing` as the first one works on all platforms :slightly_smiling_face:

Reply:
Yeah, that would be better